components/popover/popover.tsx

```diff
@@ -35,20 +40,48 @@ const Popover: React.FC<React.PropsWithChildren<PopoverProps>> = ({
   ...props
 }) => {
   const theme = useTheme()
+  const titleNode = title && useMemo(() => getReactNode(title), [title])
   const textNode = useMemo(() => getReactNode(content), [content])
+  props.color = props.color || 'lite'
 
   return (
     <Tooltip
-      text={textNode}
+      text={
+        <div>
+          {title && (
+            <div className="title">
+              <PopoverItem title line={!notSeperateTitle}>
+                {titleNode}
+              </PopoverItem>
+            </div>
+          )}
+          <div className="items">{textNode}</div>
+        </div>
+      }
       trigger={trigger}
       placement={placement}
       portalClassName={`popover ${portalClassName}`}
       {...props}>
       {children}
       <style jsx>{`
+        :global(.tooltip-content.popover) {
+          filter: drop-shadow(0px 4px 8px rgba(0, 0, 0, 0.08));
```

Comment:
can that use shadow variant？

Reply:
No, it can't https://github.com/Conflux-Chain/react-ui/blob/ad0568e4ef94d4639f5747ff2881db590d1e2961/components/styles/themes/default.ts#L93

Reply:
`:global` is also my concern

Reply:
done